Makefile

```diff
@@ -48,7 +48,7 @@ endif
 # Image build parameters
 # This tag of the image must be changed when pushed to a public repository.
 ifeq ($(REGISTRY),)
-REGISTRY := yunikorn
+REGISTRY := apache
```

Comment:
With this registry change we need to make sure that will cleanup the other images, otherwise it may cause a lot of misunderstandings.

Reply:
Yes. We should clean up the non-apache one. Once we move everything to the apache repo.

Reply:
@kingamarton please file a jira for that so we keep track and clean up

Reply:
Created YUNIKORN-154 for this purpose.